coralproject/talk
# [CORL-234] Show user's account history in the user drawer

## What does this PR do?

Adds an account history tab to the user drawer.

## How do I test this PR?

- Head to the moderate section signed in as a moderate level user
- Click on a username on a moderate card or in the community section
- Click on the account history tab in the user drawer
- See history of bans

Notes:

- Suspension records will not exist as we can't create suspensions yet
